### ISSUE TITLE
Heatsink mounting

### ISSUE THREAD

**Issue by nkrackow** [sinara-hw/Driver#20]
The current heatsink solution is not really ideal yet for mounting. The PCB spacers take a lot of force in the horizontal direction and the fins of the heatsink inhibit correct hole placement. Note that the spacers in the back are not connected to the PCB.
![7a05d0c6-0863-4024-bc87-8b73b2057526](https://user-images.githubusercontent.com/17088194/235646612-4c463a32-f716-43bb-831e-c67d711ff884.jpg)

**Comment by gkasprow** [comment]
@jmoskwa did you design this heatsink in CAD or just took a piece of raw extrusion and simply drilled the holes?

**Comment by filipswit** [comment]
It's Bartek's heatsink for RnD purpose, since we still don't have final one.

**Comment by jmoskwa** [comment]
Yes, I designed it in CAD in the first place. The vertical distance between mounting holes was from th begening 30mm.
Here's the screen from the 3D model. On top the heatsing, beneath - aluminum plate, that was attached to th PCB.
![image](https://user-images.githubusercontent.com/105804902/236149145-d4e7d26e-f020-4500-b184-61afa09d7713.png)

**Comment by gkasprow** [comment]
Did sb calculate the required thermal resistance?
